### DISCUSSION TITLE
Is grbl support “Multi-axis circular interpolation“ and “line interpolation”

### DISCUSSION THREAD

**Discussion by kongnq** [grblHAL/core#460] [General]
Hello,

we are doing one PLC (programmable logic controller). We exepct it can support “Multi-axis circular interpolation“ and “line interpolation”？   

We prefer C/C++ pogramming language。 

can it support CAM  or Gear?

Thanks  alot!

**Comment by terjeio** [comment]
> We exepct it can support “Multi-axis circular interpolation“ and “line interpolation”？

Correct.

> can it support CAM or Gear?

CAM = Computer Aided Manufacturing? Then yes provided that a suitable post-processor is available/can be made.

Gear is [this program](https://www.cnccookbook.com/gear-design-software-for-cnc-machines/)?
